server: /biclops_point_head_tracker
target_frame: marker_frame
pointing_axis: [0, 0, 1]
min_duration: 1.0
max_velocity: 0.0